server/server.go

```diff
@@ -11,14 +16,40 @@ type ContentResponseMeta struct {
 }
 
 type Content struct {
-	Body string `json:"body"`
+	ContentBody string `json:"content_body"`
 }
 
 type ContentResponse struct {
 	Meta ContentResponseMeta `json:"meta"`
 	Data []Content `json:"data"`
 }
 
+func requestFacebook(c echo.Context) error {
+  postURL := fmt.Sprintf("https://graph.facebook.com/%s/feed", FB_PAGE_ID)
+
+  postBody, _ := json.Marshal(map[string]string{
+    "message":  "Hello from Go!",
+    "access_token": FB_PAGE_ACCESS_TOKEN,
+  })
+  reqBody := bytes.NewBuffer(postBody)
+
+  resp, err := http.Post(postURL, "application/json", reqBody)
```

Comment:
is `resp` a named variable the `http.POST` method returns?  If not, you should feel no need to abbreviate.  Abbreviations lead to less readable code.  Descriptive variables will always be for the best.